tests/integration/test_charm.py

```diff
@@ -196,6 +196,9 @@ async def test_saml_login(  # pylint: disable=too-many-locals,too-many-arguments
     act: add an admin user and enable force-https mode.
     assert: user can login discourse using SAML Authentication.
     """
+    saml_app = await model.deploy("saml-integrator", channel="edge", series="jammy", trust=True, config={"entity_id": "https://login.staging.ubuntu.com", "metadata_url": "https://login.staging.ubuntu.com/saml/metadata"})
```

Comment:
I would rather use the SamlK8sTestHelper (see here: https://github.com/canonical/synapse-operator/blob/main/tests/integration/test_nginx.py#L44) instead of relying on our infra for these tests

Reply:
Hi, if you would like to use that, you might need to [add `saml-test-idp` to the integration dependencies](https://github.com/canonical/synapse-operator/blob/60b1f0bc7e754cbe39b3f32f2b17cfde25e55fc8/tox.ini#L128). There are some documents about how to use `saml-test-idp` in here https://github.com/canonical/saml-test-idp, please let me know if you have any questions 😀